modules/ROOT/pages/6.3-release-notes.adoc

```diff
@@ -0,0 +1,697 @@
+
+In previous versions of {productname} the accessibility checker used the same triangle warning icon for all alerts when displayed. In {productname} 6.3, each alert type now displays a unique icon.
+
+* Warning alerts now display as an exclamation mark nested inside a yellow circle.
+* Error alerts now display as an exclamation mark nested inside a red triangle.
+* Success alerts now display as a green tick.
+
+For information on the **Accessibility Checker** plugin, see: xref:a11ychecker.adoc[Accessibility Checker].
+
+=== Checklist 2.0.2
+
+The {productname} 6.3 release includes an accompanying release of the **Checklist** premium plugin.
+
+**Checklist** 2.0.2 includes one fix:
+
+==== Checklist toggling did not preserve selection when used on an existing list.
+
+In previous {productname} versions, when selecting content over either an ordered or unordered list, then switching the list to a checklist, the user’s content selection was not preserved.
+
+In {productname} 6.3, the user selection is now preserved when toggling a list to a checklist.
+
+For information on the **Checklist** plugin, see xref:checklist.adoc[Checklist].
+
+=== Comments 3.2.0
+
+The {productname} 6.3 release includes an accompanying release of the **Comments** premium plugin.
+
+**Comments** 3.2.0 includes one fix and one change:
+
+==== A comment could be inserted when in an empty editor or block
+
+In previous {productname} versions a bug existed that allowed comments to be placed into an empty editor or block.
+
+With this update, the comments button, menu-item, comment sidebar and shortcut keys are now disabled when the editor contains no content.
+
+==== The selection now expands to encompass the word containing the insertion point when a comment is added
+
+{productname} 6.3 adds xref:additions[two new API functions] — `tinymce.selection` and `tinymce.dom.RangeUtils`.
+
+With these two functions available, if there is no selected text when a new comment is added, {productname} automatically extends the selection to encompass the word containing the insertion point and links the newly added comment to this automatically selected word.
+
+For information on the **Comments** plugin, see: xref:introduction-to-tiny-comments.adoc[Introduction to Tiny Comments].
+
+
+=== Merge tags 1.1.0
+
+The {productname} 6.3 release includes an accompanying release of the **Merge tags** premium plugin.
+
+**Merge Tags** 1.1.0 includes one addition, one fix, and one change:
+
+==== New mceMergeTag command to wrap selected content in a merge tag
+
+A new command, `mceMergeTag`, is now available as a Text Patterns option.
+
+When this command is added as a `text_pattern`, manually typing a complete Merge Tag, including the prefix and suffix strings, into a {productname} document will result in the entered string being turned into an active merge tag when the string is completed by having a word boundary character added immediately after it.
```

Comment:
This seems overloaded complex sentence. Consider separating it into several simple parts. For example:

> Previously only a predefined list of merge tags could be used. With the new `mceMergeTag` command any selected content can be converted into a merge tag. Also configuring the inline text pattern with this command allows entering merge tags during the manual typing. Sequentially typing the prefix, some content, suffix, and word boundary character immediately wraps that content in a complete merge tag.

Reply:
To quote my commit message:

> @vpyshnenko’s re-write (with some minor edits) is now the introductory material describing the new `mceMergeTag` command.